common/src/main/java/com/wynntils/features/user/InfoMessageFilterFeature.java

```diff
@@ -106,6 +108,9 @@ public class InfoMessageFilterFeature extends UserFeature {
     @Config
     private FilterType craftedDurability = FilterType.REDIRECT;
 
+    @Config
+    private FilterType notEnoughMana = FilterType.REDIRECT;
+
     @SubscribeEvent
     public void onInfoMessage(ChatMessageReceivedEvent e) {
         if (e.getRecipientType() != RecipientType.INFO) return;
```

Comment:
Not strictly this pr, but we need to filter dialogue messages from this check and spamming queue

Reply:
There is already an issue for it

Reply:
https://github.com/Wynntils/Artemis/issues/261